WIP 3.0 Chapters/Chapter 6 Spells

```diff
@@ -1867,11 +1867,11 @@
   *3rd-level evocation*
   ___
   - **Casting Time:** 1 action
-  - **Range:** 30 feet (15-feet between targets)
+  - **Range:** 30 feet
   - **Components:** V, S
   - **Duration:** Instantaneous
   ___
-  A beam of restorative energies surges through a friendly creatures. Choose a target you can see within range, then choose a target you can see within range of it, and another within range of the second target. You can't choose the same target more than once. Each target chosen by this spell regains a number of hit points equal to 2d4 + your spellcasting ability modifier. This spell has no effect on undead or constructs.
+  A beam of restorative energies surges through a friendly creatures. Choose a target you can see within range, then choose a second target you can see within 15 feet of the first one, and a third within 15 feet range of the second target. You can't choose the same target more than once. Each target chosen by this spell regains a number of hit points equal to {@dice 2d4} + your spellcasting ability modifier. This spell has no effect on undead or constructs.
```

Comment:
I want to close this PR and fold it into the HHB 3.1 spellcasting changes. 

Is the @dice calculation here intentional, or is it maybe a carry-over from 5etools?